### PR TITLE
Bump dependencies

### PR DESCRIPTION
**Pull request by kornelski**

### CODE REVIEW

File: cargo-crev/Cargo.toml [hunk 1]
```diff
@@ -21,7 +21,7 @@ path = "src/lib.rs"
 
 [dependencies]
 atty = "0.2"
-cargo = "0.35"
+cargo = "0.38"
```
Comment (dpc) [inline cargo-crev/Cargo.toml]: There was some problem with `cargo` ...

Comment (dpc) [reply cargo-crev/Cargo.toml]: https://github.com/rust-lang/cargo/issues/7105

Comment (kornelski) [reply cargo-crev/Cargo.toml]: Couple of calls changed args.

Comment (dpc) [reply cargo-crev/Cargo.toml]: Please just double check if it's OK.

Comment (kornelski) [reply cargo-crev/Cargo.toml]: The call to acquire_package_cache_lock does the trick

File: cargo-crev/src/repo.rs [hunk 2]
```diff
@@ -152,6 +152,7 @@ impl Repo {
         let version_str = version.map(ToString::to_string);
         let dependency_request =
             Dependency::parse_no_deprecated(name, version_str.as_deref(), source.source_id())?;
+        let _lock = self.config.acquire_package_cache_lock()?;
```
Comment (dpc) [inline cargo-crev/src/repo.rs]: That's it? One place? Hmmm.... OK...

Comment (kornelski) [reply cargo-crev/src/repo.rs]: If there are more, it's only a matter of adding it where panic backtrace points to.

**Pull request merged**